spec/middleman-prismic/preview_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+require "middleman-prismic/preview"
+
+describe Middleman::Prismic::Preview do
+  it "passes through requests outside of /preview" do
+    app = rack_app
+    allow(Kernel).to receive(:system)
+    env = stub_env("http://example.com/")
+
+    middleware = Middleman::Prismic::Preview.new(app)
+    code, env, message = middleware.call env
+
+    expect(Kernel).not_to have_received(:system)
+    expect(code).to eq 200
+    expect(message).to respond_to(:each)
```

Comment:
This test could pass if the middleware didn't pass through the request, but still returned any 200.

What do you think about stubbing `app#call` so you can verify the app was called, and also verify that the response was the exact response from the app?

Reply:
resolved in 204234a